### ISSUE TITLE
Finish at least 1 Github tutorial

### ISSUE THREAD

**Issue by jonleung** [hackedu/hackedu#16]
It would also be great if you could leave a review of which git tutorials you found helpful and not helpful!

**Comment by zachlatta** [comment]
What constitutes a GitHub tutorial? "GitHub" can mean a lot of things.

**Comment by MaxWofford** [comment]
@jonleung If you are looking for a git tutorial, I'd highly recommend [git for ages 4 and up](https://www.youtube.com/watch?v=1ffBJ4sVUb4)

**Comment by zachlatta** [comment]
Git for Ages 4 and Up is a great intro to how git works. @MaxWofford, do
you think it's a good introduction to practically using git?

--
Zach Latta Executive Director of hackEDU[1] http://zachlatta.com
https://github.com/zachlatta https://twitter.com/zachlatta


On Sat, May 2, 2015, at 11:36 PM, Max Wofford wrote:
> @jonleung[2] If you are looking for a git tutorial, I'd highly
> recommend git for ages 4 and up[3]


> — Reply to this email directly or view it on GitHub[4].

>



Links:

  1. https://hackedu.us/
  2. https://github.com/jonleung
  3. https://www.youtube.com/watch?v=1ffBJ4sVUb4
  4. https://github.com/hackedu/hackedu/issues/16#issuecomment-98441741

**Comment by MaxWofford** [comment]
I learned git basics (push, pull, add, commit, checkout) through using it for personal projects, and then got an understanding of how git works by watching Git for Ages 4 and Up. Once I had that understanding learning things like how to properly rebase, reset, etc. was pretty quick.

**Comment by jonleung** [comment]
@amandadoan, any updates on this : )
ᐧ


—
Sent from my mobile–apologies for brevity and any typos!

hackedu.us | 610-761-0083

**Comment by amandadoan** [comment]
Hey guys! So the course that I used was "How to Use Git and Github" on Udacity linked here:
https://www.udacity.com/course/how-to-use-git-and-github--ud775
It offered "prerequisites" on how to navigate Bash, which was incredibly helpful since I forgot everything after not using it for a while! Also linked here:
http://linuxsurvival.com/wp/?page_id=5&id=0
Nice for a refresher or if you're a beginner.
Git for Ages 4 and Up worked fine, but I think I prefer Udacity better since it was more interactive and I wasn't just following along.

Also, I was a little iffy about these sprints at first because I kind of thought that it wasn't necessary, since anybody could just do whatever they want to do on their own time if they're motivated enough, but that's the thing. They aren't! I've been wanting to learn how to use Git for a looooong time now, and this was the motivation I needed to finally do it. These sprints are amazzzzing!
